src/features/search/components/SearchHeader/SearchHeader.web.test.tsx

```diff
@@ -124,10 +125,45 @@ describe('SearchHeader component', () => {
     await act(async () => {
       await userEvent.tab()
       await userEvent.tab()
+      await userEvent.tab()
     })
 
     const locationFilterButton = screen.getByTestId('Me localiser')
 
     expect(locationFilterButton).toHaveFocus()
   })
+
+  it('should show Rechercher when no params is given', async () => {
+    useFeatureFlagSpy.mockReturnValueOnce(true)
+    useRoute.mockReturnValueOnce({ params: { view: SearchView.Landing } })
+
+    render(
+      <SearchHeader
+        searchInputID={searchInputID}
+        addSearchHistory={jest.fn()}
+        searchInHistory={jest.fn()}
+      />
+    )
+    await act(async () => {})
+
+    expect(screen.getByText('Rechercher')).toBeTruthy()
+  })
+
+  it('should show a pin icon when no params is given', async () => {
+    useFeatureFlagSpy.mockReturnValueOnce(true)
+    useRoute.mockReturnValueOnce({ params: { view: SearchView.Landing } })
+
+    render(
+      <SearchHeader
+        searchInputID={searchInputID}
+        addSearchHistory={jest.fn()}
+        searchInHistory={jest.fn()}
+      />
+    )
+
+    await act(async () => {})
+    const title = within(screen.getByTestId('SearchHeaderTitleContainer'))
+
+    expect(title.getByTestId('location pointer not filled')).toBeTruthy()
```

Comment:
tu peux m'expliquer pour quoi le within, tu ne pouvais pas faire directement ?: screen.getByTestId('location pointer not filled'

Reply:
Simple, je veux être sûr qu'il est bien dans le titre, et pour ça je ne peux pas juste le vérifier dans l'ensemble de l'écran.

Je pense d'ailleurs que ça devrait être fait plus souvent de resserrer un peu le contexte.

Reply:
ah ok ok top merci et juste le titre pas moyen de le mock pour faire un getByText et non getByTestId ?

Reply:
Je vais essayer, car tu as raison, mais on est sur "Rechercher" et donc ça risque de se retrouver à un autre endroit :(
Ou avec un rôle peut-être !

Reply:
Ah mais en fait non je peux pas, enfin pas comme ça, car je dois prendre un conteneur. C'est pas vraiment en respect des paradigme de RNTL mais d'un autre côté ce sont des zones nons matérialisées :(